ecosystem/sep-0045.md

```diff
@@ -0,0 +1,431 @@
+the **Server** domain.
+
+#### Request
+
+```
+GET <WEB_AUTH_ENDPOINT_SEP0045>
+```
+
+##### Request Parameters:
+
+| Name            | Type          | Description                                                                                                                                                                                                                                                                                                                                                                                                                |
+| --------------- | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `address`       | `C...` string | The **Client Account** address (`C...`) that the **Client** wishes to authenticate with the **Server**.                                                                                                                                                                                                                                                                                                                    |
+| `home_domain`   | string        | A **Home Domain**. Servers that generate tokens for multiple **Home Domain**s can use this parameter to identify which home domain the **Client** hopes to authenticate with.                                                                                                                                                                                                                                              |
+| `client_domain` | string        | (optional) a **Client Domain**. Supplied by **Clients** that intend to verify their domain in addition to the **Client Account**. See [Verifying the Client Domain](#verifying-the-client-domain). **Servers** should ignore this parameter if the **Server** does not support **Client Domain** verification, or the **Server** does not support verification for the specific **Client Domain** included in the request. |
+
+Example:
+
+```
+GET https://auth.example.com/?address=CCIBUCGPOHWMMMFPFTDWBSVHQRT4DIBJ7AD6BZJYDITBK2LCVBYW7HUQ&home_domain=example.com
+```
+
+#### Response
+
+##### Success
+
+On success the endpoint must return `200 OK` HTTP status code and a JSON object with the following fields:
+
+- `authorization_entries`: a list of XDR-encoded `SorobanAuthorizationEntry`. It contains an entry for each of the
+  **Client Account**'s signers in addition to a signed entry from the **Home Domain Address** and optionally the
+  **Client Domain Address**. The entries are not guaranteed to be in any specific order.
```

Comment:
> an entry for each of the  **Client Account**'s signers

I'm confused by this. If the client account is a contract (which it must be in SEP-45), then how does the server know whether the account has multiple signers, one signer, or no signers at all?

Reply:
You're right, it does not know. At most, there would be one auth entry, independent of the account's number of signers. I'll fix this.